loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/CheckFlowsTool.java

```diff
@@ -0,0 +1,134 @@
+import eu.itesla_project.computation.ComputationManager;
+import eu.itesla_project.computation.local.LocalComputationManager;
+import eu.itesla_project.iidm.import_.Importers;
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.StateManager;
+import eu.itesla_project.iidm.network.util.Networks;
+import eu.itesla_project.loadflow.api.LoadFlow;
+import eu.itesla_project.loadflow.api.LoadFlowFactory;
+import eu.itesla_project.loadflow.api.LoadFlowParameters;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.it>
+ */
+@AutoService(Tool.class)
+public class CheckFlowsTool implements Tool {
+
+    private static Command COMMAND = new Command() {
+
+        @Override
+        public String getName() {
+            return "check-flows";
+        }
+
+        @Override
+        public String getTheme() {
+            return "Loadflow Validation";
+        }
+
+        @Override
+        public String getDescription() {
+            return "Check flows of a network";
+        }
+
+        @Override
+        public Options getOptions() {
+            Options options = new Options();
+            options.addOption(Option.builder().longOpt("case-file")
+                    .desc("case file path")
+                    .hasArg()
+                    .argName("FILE")
+                    .required()
+                    .build());
+            options.addOption(Option.builder().longOpt("output-file")
+                    .desc("output file path")
+                    .hasArg()
+                    .argName("FILE")
+                    .required()
+                    .build());
+            options.addOption(Option.builder().longOpt("load-flow")
+                    .desc("run loadflow")
+                    .build());
+            options.addOption(Option.builder().longOpt("verbose")
+                    .desc("verbose output (true/false)")
+                    .hasArg()
+                    .argName("TRUE/FALSE")
+                    .build());
+            return options;
+        }
+
+        @Override
+        public String getUsageFooter() {
+            return null;
+        }
+
+    };
+
+    @Override
+    public Command getCommand() {
+        return COMMAND;
+    }
+
+    @Override
+    public void run(CommandLine line) throws Exception {
+        Path caseFile = Paths.get(line.getOptionValue("case-file"));
+        Path outputFile = Paths.get(line.getOptionValue("output-file"));
+        CheckFlowsConfig config = CheckFlowsConfig.load();
+        if (line.hasOption("verbose")) {
+            config.setVerbose(Boolean.parseBoolean(line.getOptionValue("verbose")));
+        }
+        if (Files.isRegularFile(caseFile)) {
+            System.out.println("Loading case " + caseFile);
+            Network network = Importers.loadNetwork(caseFile);
+            if (network == null) {
+                throw new RuntimeException("Case " + caseFile + " not found");
+            }
```

Comment:
Is the Files.isRegularFile(...) test is really necessary? This should be handled by loadNetwork method.